site/source/conf.py

```diff
@@ -70,14 +70,15 @@
 TAGS = []
 smv_tag_whitelist = multiversion_regex_builder(TAGS)
 # Whitelist pattern for branches (set to None to ignore all branches)
-BRANCHES = ["0.2.x", "0.3.1.x", "0.3.2.x", "master"]
+BRANCHES = ["0.2.x", "0.3.1.x", "0.3.2.x", "0.3.4.x"]
 smv_branch_whitelist = multiversion_regex_builder(BRANCHES)
 # Defines which version is considered to be the latest stable version.
 # Must be listed in smv_tag_whitelist or smv_branch_whitelist.
 smv_latest_version = "master"
-smv_rename_latest_version = "latest"
+smv_rename_latest_version = "0.3.5.x"
```

Comment:
Also, this should probably be 0.3.4.4 in order to maximize backwards compatibility